napari_allencell_annotator/view/images_view.py

```diff
@@ -110,23 +114,20 @@ def _update_shuff_text(self, checked: bool):
             self.shuffle.setText("Shuffle and Hide")
 
     def reset_buttons(self):
+        self.enable_image_edits()
         self._toggle_delete(False)
         self._toggle_shuffle(False)
         self._update_shuff_text(False)
         self.toggle_add(True)
 
     def _delete_clicked(self):
         if len(self.file_widget.checked) > 0:
-            msg_box = QMessageBox()
-            msg: str = "Are you sure you want to delete these files?\n"
+            msg: str = "Delete these files?"
```

Comment:
really like the addition of these message boxes. Might want to make this a bit more explicit in its message